docker/include/bin/start_faux

```diff
@@ -141,6 +141,26 @@ if [ -n "${options[ntp_client]}" ]; then
     java -jar NTPClient/build/libs/NTPClient-1.0-SNAPSHOT.jar "time.google.com" "123" "3" &
 fi
 
+# ntp_pass queries the NTP server learnt from DHCP. ntp_fail sends to time.google.com
+if [ -n "${options[ntp_pass]}" ] || [ -n "${options[ntp_fail]}" ]; then
+    (while date; do
+        if [ -n "${options[ntp_pass]}" ]; then
+            dhcp_ntp=$(fgrep NTPSERVERS= /run/ntpdate.dhcp)
+            if [ -n "${dhcp_ntp}" ]; then
+                ntp_server=`echo $dhcp_ntp | cut -d "'" -f 2`
+                echo NTP server from DHCP $ntp_server
+            fi
+        elif [ -n "${options[ntp_fail]}" ]; then
+            ntp_server=time.google.com
+        fi
+        if [ -n "${ntp_server}" ]; then
```

Comment:
if statement is redundant, there is no case when it can get here without this being defined

Reply:
updated, thanks.